### PR TITLE
DOC-2597: Add numeric functions category and pi doc

### LINKED ISSUE #2597: [DOC] Add function `PI()` to  `Numeric Functions` Category
**Summary**

https://github.com/datafuselabs/databend/pull/2584

### PR DESCRIPTION
**Pull request by BohuTANG**
I hereby agree to the terms of the CLA available at: https://databend.rs/policies/cla/

## Summary

Add numeric functions category and pi doc

## Changelog

- Documentation

## Related Issues

Fixes #2597 

## Test Plan

Unit Tests

Stateless Tests

### CODE REVIEW

File: website/databend/docs/sqlstatement/numeric-functions/pi.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+Returns the value of pi as a floating-point value.
+
+## Syntax
+
+```sql
+PI()
+```
+
+## Arguments
```
Comment (PsiACE) [inline website/databend/docs/sqlstatement/numeric-functions/pi.md:14]: If no arguments is required, we need to point it or remove this section

Comment (BohuTANG) [reply website/databend/docs/sqlstatement/numeric-functions/pi.md:14]: This is a function template but the arguments is NULL.

Comment (PsiACE) [reply website/databend/docs/sqlstatement/numeric-functions/pi.md:14]: :ok_hand:

**Pull request merged**